eq-author/src/App/AccessDeniedPage/index.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+
+import BaseLayout from "components/BaseLayout";
+import { Grid, Column } from "components/Grid";
+import Link from "components/Link";
+import icon from "./icon-denied.svg";
+import styled from "styled-components";
+import { Titled } from "react-titled";
+
+const CenteredPane = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding-top: 6em;
+`;
+
+const Icon = styled.div`
+  background: url(${icon}) no-repeat center center;
+  background-size: contain;
+  width: 100px;
+  height: 100px;
+  margin-bottom: 1em;
+`;
+
+const Title = styled.h1`
+  font-size: 1em;
+  font-weight: 600;
+  margin: 0;
+`;
+
+const AccessDenied = () => (
+  <BaseLayout>
```

Comment:
This should use the new `Layout` component so won't need a lot of the components.

Reply:
Nice! have changed it now.